packages/legacy/core/App/components/misc/ScanCamera.tsx

```diff
@@ -12,13 +12,13 @@ interface Props {
   torchActive?: boolean
 }
 const ScanCamera: React.FC<Props> = ({ handleCodeScan, error, enableCameraOnError, torchActive }) => {
-  const [orientation, setOrientation] = useState('PORTRAIT')
+  const [orientation, setOrientation] = useState(OrientationType.PORTRAIT)
   const [cameraActive, setCameraActive] = useState(true)
   const orientationDegrees: { [key: string]: string } = {
-    PORTRAIT: '0deg',
-    'LANDSCAPE-LEFT': '270deg',
-    'PORTRAIT-UPSIDEDOWN': '180deg',
-    'LANDSCAPE-RIGHT': '90deg',
+    [OrientationType.PORTRAIT]: '0deg',
+    [OrientationType['LANDSCAPE-LEFT']]: '270deg',
+    [OrientationType['PORTRAIT-UPSIDEDOWN']]: '180deg',
```

Comment:
Is one of these duplicated and we're missing landscape left now? Or am I confused

Reply:
all 4 orientations are there. I don't see what you're referring to with duplicate/missing entries

Reply:
right I accidentally duplicated portrait upside-down when I was updating things. Fixed now